server/controllers/articles.js

```diff
@@ -214,6 +219,58 @@ class ArticleController {
       next(err);
     }
   }
+
+  /**
+   * Get article ratings
+   * @async
+   * @param  {object} req - Request object
+   * @param {object} res - Response object
+   * @param {object} next The next middleware
+   * @return {json} Returns json object
+   * @static
+   */
+  static async getArticleRatings(req, res, next) {
+    try {
+      const { slug } = req.params;
+      const { page, search } = req.query;
+      const paginate = new Pagination(page, req.query.limit);
+      const { limit, offset } = await paginate.getQueryMetadata();
+
+      // Get the articleId
+      const article = await Article.findOne({ where: { slug } });
+
+      if (!article) return Response.error(res, 404, 'Article does not exist');
+
+      const { id: articleId } = article.dataValues;
+
+      const ratings = await Rating.findAndCountAll({
```

Comment:
unfortunately, the `rating count` you are using here can't fly or make much sense.

if you want a count, you need to run a separate count query without the `limit` and `offset` but should include the `where clause`

Reply:
I'm on it boss